cli/lsp/cache.rs

```diff
@@ -1,123 +1,14 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
-use crate::args::ConfigFile;
-use crate::args::Flags;
-use crate::cache::FetchCacher;
-use crate::graph_util::graph_valid;
 use crate::http_cache;
-use crate::proc_state::ProcState;
-use crate::resolver::ImportMapResolver;
-use crate::resolver::JsxResolver;
 
-use deno_core::anyhow::anyhow;
-use deno_core::error::AnyError;
 use deno_core::parking_lot::Mutex;
 use deno_core::ModuleSpecifier;
-use deno_runtime::permissions::Permissions;
-use deno_runtime::tokio_util::create_basic_runtime;
-use import_map::ImportMap;
 use std::collections::HashMap;
 use std::fs;
 use std::path::Path;
-use std::path::PathBuf;
 use std::sync::Arc;
-use std::thread;
 use std::time::SystemTime;
-use tokio::sync::mpsc;
-use tokio::sync::oneshot;
-
-type Request = (
-  Vec<(ModuleSpecifier, deno_graph::ModuleKind)>,
-  oneshot::Sender<Result<(), AnyError>>,
-);
-
-/// A "server" that handles requests from the language server to cache modules
-/// in its own thread.
-#[derive(Debug)]
-pub struct CacheServer(mpsc::UnboundedSender<Request>);
```

Comment:
I don't think we need this. If the user requests to cache, we can create a graph each time.